include/image-commands.mk

```diff
@@ -272,8 +272,10 @@ define Build/jffs2
 endef
 
 define Build/kernel2minor
-	kernel2minor -k $@ -r $@.new $(1)
-	mv $@.new $@
+	$(eval temp_file := $(shell mktemp))
+	cp $@ $(temp_file)
+	kernel2minor -k $(temp_file) -r $(temp_file).new $(1)
+	mv $(temp_file).new $@
 endef
```

Comment:
maybe you should also remove the tmp file again? so
```
rm $(temp_file)
```

Reply:
Done, as suggested.

Reply:
I'm unsure about the '-f' option. Why? :S

Reply:
okay! :)